### DISCUSSION TITLE
Tooltip gets hanging when removed from a dynamic list of items

### DISCUSSION THREAD

**Discussion by jmfk** [google/mesop#1117] [Q&A]
I have a list of projects that each have a delete button with a Tooltip.
But when I press the button (the tooltip is active and showing) the text for the Tooltip gets hanging like a "ghost" text. It gets removed correct if the page reloads. It seems like since the definition of the Tooltip is removed it can't be cleaned up.

Is there a way to solve this? Is this a bug (seems like a bug too me)?

**Comment by richard-to** [comment]
I haven't had a chance to reproduce the issue it yet, but yes, this sounds like a bug most likely. Feel free to create an Bug in the issue tracker.

**Comment by jmfk** [comment]
I’m do that! Thanks Skickat från min iPhone27 nov. 2024 kl. 16:37 skrev Richard To ***@***.***>:﻿
I haven't had a chance to reproduce the issue it yet, but yes, this sounds like a bug most likely. Feel free to create an Bug in the issue tracker.
